app/src/main/java/com/starstruckstech/pgpodandroid/CentralLogger.kt

```diff
@@ -0,0 +1,19 @@
+package com.starstruckstech.pgpodandroid
+
+import com.google.firebase.firestore.FirebaseFirestore
+
+object CentralLogger {
+    private val db = FirebaseFirestore.getInstance()
+
+    fun log(level: LogLevel, tag: String, message: String) {
+        val logEntry = LogEntry(level, tag, message)
```

Comment:
This should be implemented using Timber. You need to have 3 functions for now :
logWarning
logError
logStatement